bin/wasm-node/javascript/demo/demo.js

```diff
@@ -38,6 +38,7 @@ const client = smoldot.start({
     forbidWs: false,
     forbidNonLocalWs: false,
     forbidWss: false,
+    cpuRateLimit: 0.5,
```

Comment:
Note: I've intentionally left this in the demo so that it mimics what substrate-connect will now do (https://github.com/paritytech/substrate-connect/pull/900)